clicmd

```diff
@@ -116,7 +128,9 @@ function subcommand {
   # if command has @sudo tag - we need to run the function with elevated privileges
   if grep -q "^#\\s*@sudo\\s\\+${cmd}\\s" "${REAL_CMD}" && [[ ${EUID} -ne 0 ]]; then
     local sub_path="${cmd/#cmd_}"
-    GROUP=${GROUP} exec sudo "${USER_CMD}" "${sub_path//_/ }" "$@"
+    GROUP=${GROUP} sudo "${USER_CMD}" "${sub_path//_/ }" "$@"
+    log_audit "$?" "${USER_CMD}" "${sub_path//_/ }" "$@"
```

Comment:
It won't execute when `sudo` fail because `bash -e` is used.
Try to wrap this two lines with `set +e ... set -e`

Reply:
Or even better, use something like:
```bash
    RC=0
    GROUP=${GROUP} sudo "${USER_CMD}" "${sub_path//_/ }" "$@" || RC=$?
    log_audit "${RC}" "${USER_CMD}" "${sub_path//_/ }" "$@"
```
It looks more honest ;)

Reply:
Yeap, looks like!